<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSeznamPruvodek</class>
 <widget class="QDialog" name="DialogSeznamPruvodek">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textDnesVytistene">
     <property name="text">
      <string>Dnes vytištěné průvodky:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="seznamDnesVytistene">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <item>
      <property name="text">
       <string>(žádné dnes vytištěné průvodky)</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="sekceZaraditFronty" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="tlacitkoZaraditDoFronty">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Zařadit do fronty</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="textPruvodkyVeFronte">
     <property name="text">
      <string>Průvodky v tiskové frontě</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="seznamPruvodkyVeFronte">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>(žádné nevytištěné průvodky v tiskové frontě)</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="tlacitkoZrusit">
        <property name="toolTip">
         <string>Zrušit (Esc)</string>
        </property>
        <property name="text">
         <string>Zrušit</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tlacitkoPotvrdit">
        <property name="toolTip">
         <string>Potvrdit (Ctrl+Enter)</string>
        </property>
        <property name="text">
         <string>Potvrdit</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
